Core/MIPS/IR/IRCompVFPU.cpp

```diff
@@ -1009,9 +1020,40 @@ namespace MIPSComp {
 	}
 
 	void IRFrontend::Comp_Vmscl(MIPSOpcode op) {
-		DISABLE;
+		CONDITIONAL_DISABLE;
+
+		// Matrix scale, matrix by scalar (no prefixes)
+		// d[N,M] = s[N,M] * t[0]
+
+		int vs = _VS;
+		int vd = _VD;
+		int vt = _VT;
+
+		MatrixSize sz = GetMtxSize(op);
+		if (sz != M_4x4) {
+			DISABLE;
+		}
+		if (GetMtx(vt) == GetMtx(vd)) {
+			DISABLE;
+		}
+		int n = GetMatrixSide(sz);
+
+		// The entire matrix is scaled equally, so transpose doesn't matter.  Let's normalize.
```

Comment:
Ah yes, oops.  Looks like a perfect place for our "transpose matrix" op, could apply it almost like a D prefix and it could map to a ST4.

-[Unknown]